src/components/Header/Menu/Menu.component.jsx

```diff
@@ -0,0 +1,53 @@
+
+import React from 'react';
+import styled from 'styled-components';
+
+
+const MenuButton  = styled.button`
```

Comment:
This component seems to be duplicated from `src/components/Header/LoginControl/LoginControl.component.jsx`